lib/ruby_lsp/requests/completion_resolve.rb

```diff
@@ -56,8 +56,16 @@ def perform
           end
         end
 
+        first_entry = T.must(entries.first)
+
+        if first_entry.is_a?(RubyIndexer::Entry::Member)
+          detail = "(#{first_entry.parameters.map(&:decorated_name).join(", ")})"
```

Comment:
Worth extracting `.parameters.map(&:decorated_name).join(", ")` be a method on `Entry`?

Reply:
Definitely! We used the same code in other places.